### PR TITLE
Bugfixes/post v1 aug 2020

### PR DESCRIPTION
**Pull request by rickycodes**
**Description**

This is a grouping of PRs to clean things up a bit and make review/QA a bit easier on everyone.
this PR includes the following work:

#1781 (this has been approved, this is just a code change, no functionality changed)
#1759 (this is a bugfix and needs QA)
#1728 (just some code cleanup, no functionality changed)
#1787 (add missing currencies, in particular SAI and INR needs QA)
#1800 (minor improvement to search input, needs QA)

### CODE REVIEW

File: app/components/Views/AccountBackupStep1/index.js [hunk 1]
```diff
@@ -228,10 +235,7 @@ const AccountBackupStep1 = props => {
 				testID={'account-backup-step-1-screen'}
 			>
 				<View style={styles.wrapper} testID={'protect-your-account-screen'}>
-					<OnboardingProgress
-						steps={['Create password', 'Secure wallet', 'Confirm seed phrase']}
-						currentStep={1}
-					/>
+					<OnboardingProgress steps={steps} currentStep={1} />
```
Comment (estebanmino) [inline app/components/Views/AccountBackupStep1/index.js:238]: is this component used in other places? if not maybe we could put the steps cont in the component itself

File: app/components/Views/AccountBackupStep1B/index.js [hunk 2]
```diff
@@ -212,6 +212,12 @@ const AccountBackupStep1B = props => {
 	const showWhatIsSeedphrase = () => setWhatIsSeedphraseModal(true);
 	const hideWhatIsSeedphrase = () => setWhatIsSeedphraseModal(false);
 
+	const steps = [
+		strings('choose_password.title'),
+		strings('choose_password.secure'),
+		strings('choose_password.confirm')
```
Comment (estebanmino) [inline app/components/Views/AccountBackupStep1B/index.js:218]: I see the same steps than before here

File: app/components/Views/ImportWallet/index.js [hunk 3]
```diff
@@ -310,30 +319,30 @@ class ImportWallet extends PureComponent {
...
-			await AsyncStorage.setItem('@MetaMask:biometryChoiceDisabled', 'true');
-			await AsyncStorage.setItem('@MetaMask:passcodeDisabled', 'true');
+			await AsyncStorage.removeItem(BIOMETRY_CHOICE);
+			await AsyncStorage.setItem(BIOMETRY_CHOICE_DISABLED, TRUE);
+			await AsyncStorage.setItem(PASSCODE_DISABLED, TRUE);
 		}
 
 		try {
-			await AsyncStorage.removeItem('@MetaMask:nextMakerReminder');
+			await AsyncStorage.removeItem(NEXT_MAKER_REMINDER);
```
Comment (estebanmino) [inline app/components/Views/ImportWallet/index.js:337]: I believe we don't need this item anymore, I erased the code for the reminder alert while ago

File: app/components/Views/AccountBackupStep1/index.js (hunk 1 above)
Comment (rickycodes) [reply app/components/Views/AccountBackupStep1/index.js]: it's used in other places, but I've de-duplicated things a bit by moving the steps variables into a new `/constants` file now

File: app/components/Views/AccountBackupStep1B/index.js (hunk 2 above)
Comment (rickycodes) [reply app/components/Views/AccountBackupStep1B/index.js]: https://github.com/MetaMask/metamask-mobile/pull/1797#discussion_r477604140

File: app/components/Views/ImportWallet/index.js (hunk 3 above)
Comment (rickycodes) [reply app/components/Views/ImportWallet/index.js:337]: It's still all over `develop` from what I am seeing :eyes: https://github.com/MetaMask/metamask-mobile/blob/master/app/components/Views/CreateWallet/index.js#L121

If you think we can delete all the `nextMakerReminder` AsyncStorage calls we probably should. I suggest we make sure and then possible do the redaction in a second pull request?

I did find one place where I missed a constant (from pedro's changes). I'm going to add that one.

**Review by estebanmino — APPROVED** [review]

**Review by ibrahimtaveras00 — CHANGES_REQUESTED** [review]
Few issues mentioned to @rickycodes

**Comment by rickycodes** [comment]
@ibrahimtaveras00 this one should be good-to-go again!

**Review by ibrahimtaveras00 — APPROVED** [review]
Fixes look good, QA Passed 👍

**Pull request merged**